readers-frontend/src/components/DesktopHeader.astro

```diff
@@ -0,0 +1,102 @@
+---
+import type { categories } from "@prisma/client";
+import prisma from "../prisma";
+import { Icon } from "astro-icon";
+import SearchInterface from "./SearchInterface.astro";
+
+interface Props {
+  active: string;
+}
+
+let menuCategories: categories[] = (await prisma.categories.findMany()).filter(
+  (category) => category.categories_showMenu
+);
+
+// No nested elements, main items. Ignore muse
+let mainDisplayItems = menuCategories.filter(
+  (category) =>
+    category.categories_nestUnder === null &&
+    category.categories_name !== "muse"
+);
+
+// Construct nested items map - saved for later
+// let nestedItems = menuCategories.reduce((accumulator, category, _) => {
+//   const parent = category.categories_nestUnder;
+//   if (parent !== null) {
+//     let subCategories = accumulator.get(parent);
+
+//     if (!subCategories) subCategories = [];
+
+//     subCategories.push(category);
+
+//     accumulator.set(parent, subCategories);
+//   }
+//   return accumulator;
+// }, new Map<number, categories[]>());
+
+const { active = "Home" } = Astro.props;
+---
+
+<header class="flex h-28 w-full">
```

Comment:
I know this is a bit annoying. Gracie replied to my comment on the design. So can you adjust the css so that we have about 250px on each side instead of the current one?

Reply:
Yeah I'll pick the closets tailwind value. I'd rather not diverge from their units cause everything fits nicely if we use theirs all over

Reply:
yeah sounds good